plugins/nodes/src/js/data/MesosMasters.js

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import { Observable } from "rxjs/Observable";
+import "rxjs/add/observable/timer";
+import "rxjs/add/operator/do";
+import "rxjs/add/operator/map";
+import "rxjs/add/operator/retry";
+import "rxjs/add/operator/filter";
+
+import MesosStateStore from "#SRC/js/stores/MesosStateStore";
+
+import LeaderGrid from "../components/LeaderGrid";
+
+function isEmptyObject(obj) {
```

Comment:
I am not sure that method is bulletproof for the util. I also saw that logic already duplicated in many places in the code. If there is a util method I would that instead of this one. If we want to have that on util, I would create a ticket for that, and then it will get the priority it deserves. Do you think that makes sense?